ix-dev/community/codegate/questions.yaml

```diff
@@ -0,0 +1,398 @@
+  - name: Labels Configuration
+    description: Configure Labels for Codegate
+  - name: Resources Configuration
+    description: Configure Resources for Codegate
+
+questions:
+  - variable: codegate
+    label: ""
+    group: Codegate Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: additional_args
+          label: Additional Arguments
+          description: Configure additional arguments for Codegate.
+          schema:
+            type: list
+            default: []
+            items:
+              - variable: arg
+                label: Argument
+                schema:
+                  type: string
+                  required: false
+        - variable: additional_envs
+          label: Additional Environment Variables
+          description: Configure additional environment variables for Codegate.
+          schema:
+            type: list
+            default: []
+            items:
+              - variable: env
+                label: Environment Variable
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: name
+                      label: Name
+                      schema:
+                        type: string
+                        required: false
+                    - variable: value
+                      label: Value
+                      schema:
+                        type: string
+                        required: false
+  - variable: run_as
+    label: ""
+    group: User and Group Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: user
+          label: User ID
+          description: The user id that Codegate files will be owned by.
+          schema:
+            type: int
+            min: 568
+            default: 568
+            required: true
+        - variable: group
+          label: Group ID
+          description: The group id that Codegate files will be owned by.
+          schema:
+            type: int
+            min: 568
+            default: 568
+            required: true
+
+  - variable: network
+    label: ""
+    group: Network Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: proxy_port
+          label: Proxy Port
+          description: The port for Codegate Proxy
+          schema:
+            type: int
+            default: 8990
+            required: true
+            $ref:
+              - definitions/port
+        - variable: api_port
+          label: API Port
+          description: The port for Codegate API
+          schema:
+            type: int
+            default: 8989
+            required: true
+            $ref:
+              - definitions/port
+        - variable: web_port
+          label: Dashboard Port
+          description: The port for Codegate Dashboard
+          schema:
+            type: int
+            default: 9090
+            required: true
+            $ref:
+              - definitions/port
```

Comment:
Please see newer apps like Umami or zigbee2mqtt for new style ports (you also need changes in docker-compose and basic-values)